clients/ui/frontend/src/app/AppRoutes.tsx

```diff
@@ -25,33 +26,34 @@ export const useAdminSettings = (): NavDataItem[] => {
   // get auth access for example set admin as true
   const isAdmin = true; //this should be a call to getting auth / role access
 
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
```

Comment:
We will remove this when auth is implemented

Reply:
Can you add a todo in the code to add that info?